### PR TITLE
Fix invalid yarn.lock

### PR DESCRIPTION
**Pull request by Palid**
---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Comment by gsouquet** [comment]
Slightly curious, what was invalid about it?

**Comment by Palid** [comment]
> Slightly curious, what was invalid about it?

Wrong package version, we have bumped `package.json` but haven't `yarn.lock`. 😄

**Review by gsouquet — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -7765,7 +7765,7 @@ matrix-react-test-utils@^0.2.3:
     "@babel/traverse" "^7.13.17"
     walk "^2.3.14"
 
-matrix-widget-api@^0.1.0-beta.16:
+matrix-widget-api@^0.1.0-beta.15, matrix-widget-api@^0.1.0-beta.16:
```
Comment (turt2live) [inline yarn.lock:7768]: This has the problem of now bundling two widget APIs which will break Jitsi calls at release. It needs to exactly match the react SDK.

**Review by turt2live — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (Palid) [reply yarn.lock:7768]: Looking into it.

**Pull request merged**